lib/modules/datasource/go/base.spec.ts

```diff
@@ -16,12 +16,14 @@ const hostRules = mocked(_hostRules);
 describe('modules/datasource/go/base', () => {
   describe('simple cases', () => {
     it.each`
-      module                           | datasource          | packageName
-      ${'gopkg.in/foo'}                | ${'github-tags'}    | ${'go-foo/foo'}
-      ${'gopkg.in/foo/bar'}            | ${'github-tags'}    | ${'foo/bar'}
-      ${'github.com/foo/bar'}          | ${'github-tags'}    | ${'foo/bar'}
-      ${'bitbucket.org/foo/bar'}       | ${'bitbucket-tags'} | ${'foo/bar'}
-      ${'code.cloudfoundry.org/lager'} | ${'github-tags'}    | ${'cloudfoundry/lager'}
+      module                                  | datasource          | packageName
+      ${'gopkg.in/foo'}                       | ${'github-tags'}    | ${'go-foo/foo'}
+      ${'gopkg.in/foo/bar'}                   | ${'github-tags'}    | ${'foo/bar'}
+      ${'github.com/foo/bar'}                 | ${'github-tags'}    | ${'foo/bar'}
+      ${'bitbucket.org/foo/bar'}              | ${'bitbucket-tags'} | ${'foo/bar'}
+      ${'code.cloudfoundry.org/lager'}        | ${'github-tags'}    | ${'cloudfoundry/lager'}
+      ${'dev.azure.com/foo/bar/_git/baz.git'} | ${'git-tags'}       | ${'https://dev.azure.com/foo/bar/_git/baz'}
+      ${'dev.azure.com/foo/bar/baz.git'}      | ${'git-tags'}       | ${'https://dev.azure.com/foo/bar/_git/baz'}
```

Comment:
Please add one test for when the split is mistakenly <4, e.g. `dev.azure.com/foo/bar.git`